src/scrapers/mizrahi.ts

```diff
@@ -260,6 +276,24 @@ class MizrahiScraper extends BaseScraperWithBrowser<ScraperSpecificCredentials>
       balance: +response.body.fields?.Yitra,
     };
   }
+
+  private shouldMarkAsPending(txn: Transaction): boolean {
+    if (this.options.optInFeatures?.includes('mizrahi:pendingIfNoIdentifier') && !txn.identifier) {
```

Comment:
make the `mizrahi:pendingIfNoIdentifier` a reusable constant

Reply:
I prefer letting TS verify the constants instead of defining consts (and enums have their own issues).
WDYT?

Reply:
I think holding a string will require me to use the find tool, and it doesn't gurentee it will find only the related items, while using a reference, even a `const`, will send me by "go to declaration" to the list of the feature.

But I'm not really arguing on such things, this is my convention, but I accept yours too.